target/linux/ramips/dts/DWR-512-B.dts

```diff
@@ -0,0 +1,120 @@
+#include "rt5350.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "ralink,rt5350-soc";
+	model = "D-Link DWR-512 B";
+
+	gpio-keys-polled {
+		compatible = "gpio-keys-polled";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		poll-interval = <20>;
+
+		wps {
+			label = "wps";
+			gpios = <&gpio0 0 GPIO_ACTIVE_LOW>;
+			linux,code = <0x211>;
```

Comment:
the linux,code line. you should use the define from dt-bindings/input/input.h instead of the hex code: https://github.com/lede-project/source/commit/2b55c83e68068887db9e49de8e81b1b4a267a4f2